.github/workflows/ci.yaml

```diff
@@ -88,7 +90,7 @@ jobs:
           fi
       - name: Run integration tests
         # set a predictable model name so it can be consumed by charm-logdump-action
-        run: sg microk8s -c "tox run -e ${{ matrix.tox-environments }} -- --model testing -m '${{ steps.select-tests.outputs.mark_expression }}'"
+        run: sg snap_microk8s -c "tox run -e ${{ matrix.tox-environments }} -- --model testing -m '${{ steps.select-tests.outputs.mark_expression }}'"
         env:
           CI_PACKED_CHARMS: ${{ needs.build.outputs.charms }}
       - name: Dump logs
```

Comment:
These changes are NOT to be merged. This PR is only to run the code on 3.1 pipelines

Reply:
([The PR to be merged](https://github.com/canonical/mongodb-k8s-operator/pull/161) is equal to the current one, except the last commit.)